torch/quantization/observer.py

```diff
@@ -750,46 +756,50 @@ def _combine_histograms(
             # remaining should go to dst_bin2
             if dst_bin_cnt < src_bin_count:
                 dst_histogram[dst_bin2] += src_bin_count - dst_bin_cnt
+        return dst_histogram
```

Comment:
This seems to be an unnecessary change.

Reply:
this is needed for compile as well, ignored functions seems to take Tensor as return only